### PR TITLE
Russian

### PR DESCRIPTION
**Pull request by mmavka**

### CODE REVIEW

File: Common/Languages/Russian/Keyed/Translations.xml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+  <PlanningExtended.Designator.PlanFloors.Label>Полы</PlanningExtended.Designator.PlanFloors.Label>
+  <PlanningExtended.Designator.PlanFloors.Desc>Разместите план для полов.\n\nЩелкните правой кнопкой мыши, чтобы выбрать одну из форм.\n\nУдерживайте {0}, чтобы выбрать уже существующий цвет.</PlanningExtended.Designator.PlanFloors.Desc>
+
+  <PlanningExtended.Designator.PlanObjects.Label>Объекты</PlanningExtended.Designator.PlanObjects.Label>
+  <PlanningExtended.Designator.PlanObjects.Desc>Разместите план для объектов.\n\nЩелкните правой кнопкой мыши, чтобы выбрать одну из форм.\n\nУдерживайте {0}, чтобы выбрать уже существующий цвет.</PlanningExtended.Designator.PlanObjects.Desc>
+
+  <PlanningExtended.Designator.PlanWalls.Label>Стены</PlanningExtended.Designator.PlanWalls.Label>
+  <PlanningExtended.Designator.PlanWalls.Desc>Разместите план для стен.\n\nЩелкните правой кнопкой мыши, чтобы выбрать одну из форм.\n\nУдерживайте {0}, чтобы выбрать уже существующий цвет.</PlanningExtended.Designator.PlanWalls.Desc>
+
```
Comment (Scherub) [inline Common/Languages/Russian/Keyed/Translations.xml:15]:
```suggestion

  <PlanningExtended.Designator.PaintPlan.Label>план покраски</PlanningExtended.Designator.PaintPlan.Label>
  <PlanningExtended.Designator.PaintPlan.Desc>Закрасьте существующие обозначения.\n\nЩелкните правой кнопкой мыши, чтобы выбрать одну из форм.\n\nУдерживайте {0}, чтобы выбрать уже существующий цвет.</PlanningExtended.Designator.PaintPlan.Desc>
  
```

File: Common/Languages/Russian/DefInjected/KeyBindingDef/KeyBindings.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<LanguageData>
+
+  <Planning_TogglePlanVisibility.label>переключить видимость плана</Planning_TogglePlanVisibility.label>
+  <Planning_Modifier.label>модификатор</Planning_Modifier.label>
```
Comment (Scherub) [inline Common/Languages/Russian/DefInjected/KeyBindingDef/KeyBindings.xml:5]:
```suggestion
  <Planning_ColorPicker.label>палитра цветов</Planning_ColorPicker.label>
  <Planning_Modifier.label>модификатор</Planning_Modifier.label>
```

File: Common/Languages/Russian/Keyed/Translations.xml [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  <PlanningExtended.Settings.DisplayChangePlanAppearanceDesignator.Label>Обозначение внешнего вида плана изменения отображения</PlanningExtended.Settings.DisplayChangePlanAppearanceDesignator.Label>
+  <PlanningExtended.Settings.DisplayChangePlanAppearanceDesignator.Desc>Определяет, будет ли видна кнопка указателя внешнего вида плана.</PlanningExtended.Settings.DisplayChangePlanAppearanceDesignator.Desc>
+
+  <PlanningExtended.Settings.ArePlansPersistent.Label>Постоянные планы</PlanningExtended.Settings.ArePlansPersistent.Label>
+  <PlanningExtended.Settings.ArePlansPersistent.Desc>Определяет, остаются ли планы после завершения задачи в той же ячейке.</PlanningExtended.Settings.ArePlansPersistent.Desc>
+
+  <PlanningExtended.Settings.UseCtrlForColorDialog.Label>Использовать Ctrl для выбора цвета</PlanningExtended.Settings.UseCtrlForColorDialog.Label>
+  <PlanningExtended.Settings.UseCtrlForColorDialog.Desc>Определяет, нужно ли нажимать Ctrl для отображения диалогового окна выбора цвета.</PlanningExtended.Settings.UseCtrlForColorDialog.Desc>
+
```
Comment (Scherub) [inline Common/Languages/Russian/Keyed/Translations.xml:94]:
```suggestion

  <PlanningExtended.Settings.UseSkipInsteadOfReplaceAsDefault.Label>Использовать «Пропустить» вместо «Заменить» по умолчанию</PlanningExtended.Settings.UseSkipInsteadOfReplaceAsDefault.Label>
  <PlanningExtended.Settings.UseSkipInsteadOfReplaceAsDefault.Desc>Если включено, обозначения по умолчанию пропускаются. В противном случае они заменяются.</PlanningExtended.Settings.UseSkipInsteadOfReplaceAsDefault.Desc>
  
```

**Review by Scherub — COMMENTED** [review]
Thank you for providing a translation for the Russian language!

There were a few entries missing. I've added those as suggestions with Google translated translations. So you probably have to translate it into proper Russian.

I would also prefer a branch within the original repository, as this makes it way easier for me to test your changes.

**Comment by mmavka** [comment]
image
how change?

**Review by Scherub — APPROVED** [review]
Looks like everything that is possible to translate got a translation. The only thing I found was, that one of the settings entries was a bit long. But other than that, well done. Thank you for your contribution.

**Pull request merged**